core-ui/src/resources/CustomResourceDefinitions/ExtensibilityStarterModal/ExtensibilityStarterModal.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button, MessageStrip } from 'fundamental-react';
+import { ModalWithForm } from 'shared/components/ModalWithForm/ModalWithForm';
+import { prettifyKind } from 'shared/utils/helpers';
+import { Link } from 'shared/components/Link/Link';
+import { Trans, useTranslation } from 'react-i18next';
+import { ExtensibilityStarterForm } from './ExtensibilityStarterForm';
+import pluralize from 'pluralize';
+
+export function ExtensibilityStarterModal({ crd }) {
+  const { t } = useTranslation();
+
+  const DocsLink = () => {
+    return (
+      <MessageStrip type="information" className="fd-margin-bottom--sm">
+        <Trans i18nKey="extensibility.starter-modal.messages.docs-info">
+          <Link
+            className="fd-link"
+            url="https://github.com/kyma-project/busola/tree/main/docs/extensibility"
+          />
+        </Trans>
+      </MessageStrip>
+    );
+  };
+
+  const ConfigMapInfo = () => {
+    const bold = { fontWeight: 'bolder' };
+
+    return (
+      <MessageStrip type="information" className="fd-margin-bottom--sm">
+        <Trans
+          i18nKey="extensibility.starter-modal.messages.configmap-info"
+          values={{ name: crd.metadata.name }}
+        >
+          <span style={bold} />
```

Comment:
Why do we have an empty `span` here?

Reply:
This empty `span`s are used by `Trans` component:
```
 <0>{{name}}</0> Config Map will be created or updated in <1>kube-public</1> Namespace.
```
The content inside `<0>` and `<1>` tags is rendered using bolded (?) spans.